mapmvt.c

```diff
@@ -278,22 +271,9 @@ int msMVTWriteFromQuery( mapObj *map, outputFormatObj *format, int sendheaders )
   void *buf;
   const char *mvt_buffer = msGetOutputFormatOption(map->outputformat, "EDGE_BUFFER", "10");
   int buffer = MS_ABS(atoi(mvt_buffer));
-  double res;
-  rectObj bbox;
   VectorTile__Tile mvt_tile = VECTOR_TILE__TILE__INIT;
   mvt_tile.layers = msSmallCalloc(map->numlayers,sizeof(VectorTile__Tile__Layer*));
 
-
```

Comment:
@sdlime, I see why you were getting alignment issues. This block of code needs to be kept, however the query.rect should be extended before the query, as is done in the WFS part: https://github.com/mapserver/mapserver/pull/5376/commits/032e5d3215d88a8519b0ef2405ff9a0c49eab998#diff-8b3f151aa0bfca8681cd277ced6c85cfR2619 . I know this isn't clean, but the buffering logic (similar to metabuffering when tiling) is essential IIRC for correct rendering.